packages/circuits/src/helpers/extractor.circom

```diff
@@ -261,6 +261,6 @@ template QRDataExtractor(maxDataLength) {
     component photoExtractor = PhotoExtractor(maxDataLength);
     photoExtractor.nDelimitedData <== nDelimitedData;
     photoExtractor.startDelimiterIndex <== delimiterIndices[photoPosition() - 1];
-    photoExtractor.endIndex <== nonPaddedDataLength - 1;
+    photoExtractor.endIndex <== nonPaddedDataLength - 65;
```

Comment:
`nonPaddedDataLength` was expected to be the length without the signature. Either works though.

Reply:
Here the 65 bytes removed are not the bytes of the signature but the bytes of the phone hash and the email hash.
Also, it should be dynamic and conditioned on the is_mobile_or_phone variable.

Reply:
Oh ok. But email and mobile hash was not part of the V2 data when I checked. Let me check again.